apps/hash-api/src/dom-shim.d.ts

```diff
@@ -0,0 +1,15 @@
+declare const window: {
```

Comment:
What do we need this for?

Reply:
The update of dependencies ensured that we no longer have DOM types inside the hash-api package once pruned (a good thing!), but `hash-isomorphic-utils` depends on `window` being defined: https://github.com/hashintel/hash/blob/7d08d9f6ef7df9f672fe9fb256049183a1e87816/libs/%40local/hash-isomorphic-utils/src/blocks.ts#L34

I didn't want to rewrite this / move it out – so instead used a shim here so that tsc no longer complains.